content/rs/administering/database-operations/alerting.md

```diff
@@ -5,29 +5,18 @@ weight: $weight
 alwaysopen: false
 categories: ["RS"]
 ---
-In the *Alerts* section of the Redis Enterprise Software database page,
-you can designate which database events trigger alert
-notifications.
+You can select alerts to receive notifications about database status.
```

Comment:
"You can select alerts to receive notifications and emails about database status."
I added the "and emails", what do you think?